go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -6583,14 +6640,30 @@ func (fbo *folderBranchOps) unstageLocked(ctx context.Context,
 	}
 
 	// let the server know we no longer have need
-	if wasUnmergedBranch {
+	if wasUnmergedBranch && pruningBehavior == doPruneBranches {
 		err = fbo.config.MDOps().PruneBranch(ctx, fbo.id(), unmergedBID)
 		if err != nil {
 			return err
 		}
+	} else {
+		jManager, err := GetJournalManager(fbo.config)
+		if err != nil {
+			return err
+		}
+
+		if tlfJournal, ok := jManager.getTLFJournal(fbo.id(), nil); ok {
+			err = tlfJournal.moveAway(ctx)
+			if err != nil {
+				return err
+			}
+		} else {
+			return errJournalNotAvailable
+		}
 	}
 
 	// now go forward in time, if possible
+	// !isUnmerged is true -> isUnmerged is false -> we are merged but
```

Comment:
Is this leftover from debugging or a real question?

Reply:
Left over from debugging, whoops